templates/network/up-bridge.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# https://raspberrypi.stackexchange.com/questions/71348/cannot-forward-from-wlan-interface-to-another-wlan-interface
+AP_INTERFACE="ap0" # ap
+SOURCE_INTERFACE="wlan0" # to router
+
+# Flush rules
+iptables -t nat -F
+iptables -F
+
+# Apply new rules
+iptables -t nat -A POSTROUTING -o $SOURCE_INTERFACE -j MASQUERADE
+iptables -A FORWARD -i $SOURCE_INTERFACE -o $AP_INTERFACE -m state --state RELATED,ESTABLISHED -j ACCEPT
+iptables -A FORWARD -i $AP_INTERFACE -j ACCEPT
+
+while /bin/true;
```

Comment:
This while will run in background because of "&" so it doesn't freeze the system